lib/mongo/collection/view/builder/find_command.rb

```diff
@@ -94,6 +94,7 @@ def specification
 
           def find_command
             document = BSON::Document.new('find' => collection.name, 'filter' => filter)
+            document[:collation] = collation if collation
```

Comment:
yes, it's called here: https://github.com/mongodb/mongo-ruby-driver/pull/788/files/3321ddd3889c17d49a534502e137e44d16404e76#diff-cf306a3295c965d0c497491ba4d63ea6R64
before the FindCommand Builder even has a chance to build the command.